raiden/network/blockchain_service.py

```diff
@@ -54,9 +56,12 @@ def __init__(
     def node_address(self) -> Address:
         return self.client.address
 
-    def block_number(self) -> int:
+    def block_number(self) -> BlockNumber:
         return self.client.block_number()
 
+    def block_hash(self) -> BlockHash:
+        return self.client.blockhash_from_blocknumber(self.block_number())
```

Comment:
I believe `block_number` is taken out of the latest block, so this is equivalent to:

```python
self.client.blockhash_from_blocknumber('latest')
```
But the above does only one JSON RPC request instead of two.

Reply:
https://github.com/raiden-network/raiden/pull/3547